### PR TITLE
Codestyle fixes for application and connection

### LINKED ISSUE #317: Align code to follow checkstyle
Checkstyle was added as part of #298.
Code should be aligned so we comply with it, once all of the code is aligned the ignoring of errors/warnings from checkstyle should be removed.

i.e remove this line from pom.xml
`<failOnViolation>false</failOnViolation>`

### PR DESCRIPTION
**Pull request by jwaeab**
Closes #317

### CODE REVIEW

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java [hunk 1]
```diff
@@ -35,67 +39,67 @@
...
 
-    public void setKeystore(String keystore)
+    public final void setKeystore(final String aKeystore)
     {
-        this.keystore = keystore;
+        this.keystore = aKeystore;
     }
 
-    public String getKeystorePassword()
+    public final String getKeystore_password()
```
Comment (masokol) [inline application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java:62]: Why are you changing to underscore here? Can't we use the same as for everything else? like keyStorePassword? It should be possible to solve parsing from/to json/yaml with annotations.

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java [hunk 2]
```diff
@@ -134,59 +134,75 @@ public DefaultRepairConfigurationProvider defaultRepairConfigurationProvider()
...
-    public JmxConnectionProvider jmxConnectionProvider(Config config) throws ConfigurationException
+    public JmxConnectionProvider jmxConnectionProvider(final Config config) throws ConfigurationException
     {
         return getJmxConnectionProvider(config, jmxSecurity::get);
     }
 
-    private static JmxConnectionProvider getJmxConnectionProvider(Config configuration,
-            Supplier<Security.JmxSecurity> securitySupplier) throws ConfigurationException
+    private static JmxConnectionProvider getJmxConnectionProvider(final Config configuration,
+                                                                  final Supplier<Security.JmxSecurity> securitySupplier
```
Comment (masokol) [inline application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java:193]: this looks a bit strange for me.

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/AbstractRepairConfigurationProvider.java [hunk 3]
```diff
@@ -24,19 +24,24 @@
...
+        return applicationContext;
+    }
 
     private final RepairConfiguration defaultRepairConfiguration;
 
-    protected AbstractRepairConfigurationProvider(ApplicationContext applicationContext)
+    protected AbstractRepairConfigurationProvider(final ApplicationContext anApplicationContext)
     {
-        this.applicationContext = applicationContext;
+        this.applicationContext = anApplicationContext;
```
[the remark this answers is not in the document]
Comment (jwaeab) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/AbstractRepairConfigurationProvider.java:38]: If the consensus is that all class should be prefixed with my*, it should probably be a separate fix. In some instances, changing the class variables seems to break tests.

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java (hunk 1 above)
Comment (jwaeab) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java:62]: To keep to "standard" of other variables named this way in the class.

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java (hunk 2 above)
Comment (masokol) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java:193]: Having ") throws ..." on a new line. I would prefer to end with ) and have throws on new line instead.

Comment (jwaeab) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/spring/BeanConfigurator.java]: Fixed

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/AbstractRepairConfigurationProvider.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (tommystendahl) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/AbstractRepairConfigurationProvider.java:38]: I usually prefer my.

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java (hunk 1 above)
Comment (tommystendahl) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java:62]: It looks odd with an underscore, camel case is preferred. But if there is a good reason for an exception I would not go out of my way to change.

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/AbstractRepairConfigurationProvider.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (jwaeab) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/AbstractRepairConfigurationProvider.java:38]: Should be a separate issue

File: application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java (hunk 1 above)
Comment (jwaeab) [reply application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/config/TLSConfig.java:62]: Will need to be reworked if changing. Separate issue.

**Pull request merged**